README.md

```diff
@@ -13,11 +13,19 @@
 [![PRs Welcome][prs-badge]][prs]
 
 Atom package to format your JavaScript, Typescript, CSS, Sass, and JSON using [Prettier](https://github.com/prettier/prettier).
-Comes with powerful optional [ESLint](http://eslint.org/) integration.
+Optionally integrates with [ESLint](http://eslint.org/), [EditorConfig](http://editorconfig.org/), and [stylelint](https://stylelint.io/).
+
+## How to use it
+
+After [installation](#installation) you are ready to use prettier in a way that suits your needs
+- Toggle 'format on save' to have prettier format your code upon saving
+- Run the command `Prettier: Format (Ctrl+Alt+F)` to format your file manually
+- By default, formatting will use your [prettier configuration](https://prettier.io/docs/en/configuration.html).
+  Otherwise it falls back to the prettier settings you chose in this plugin.
```

Comment:
This is good to have here, with the recent changes it's now straightforward to explain 👍